### PR TITLE
fix: `mention` in `Member()` being NoneType

### PR DESCRIPTION
**Pull request by B1ue-Dev**
## About

This pull request is about fixing the `mention` property in Member being None.

## Checklist

- [x] I've ran `pre-commit` to format and lint the change(s) made.
- [x] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [ ] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues) (If existent):.
  - resolves #
- I've made this pull request for/as: (check all that apply)
  - [ ] Documentation
  - [ ] Breaking change
  - [ ] New feature/enhancement
  - [x] Bugfix

### CODE REVIEW

**Review by Damego — APPROVED** [review]
LGTM!

**Comment by Damego** [comment]
~~Why I still can't merge prs?~~

File: interactions/api/models/member.py [hunk 1]
```diff
@@ -143,7 +143,7 @@ def mention(self) -> str:
         :return: The string of the mentioned member.
         :rtype: str
         """
-        return f"<@!{self.user.id}>" if self.nick else f"<@{self.user.id}>"
+        return f"<@!{self.id}>" if self.nick else f"<@{self.id}>"
```
Comment (Toricane) [inline interactions/api/models/member.py]: What about this
```suggestion
        return f"<@{'!' if self.nick else ''}{self.id}>"
```

**Review by Toricane — APPROVED** [review]
LGTM!

**Pull request merged**